api/src/main/java/org/openmrs/module/eptsreports/reporting/library/queries/BreastfeedingQueries.java

```diff
@@ -0,0 +1,39 @@
+package org.openmrs.module.eptsreports.reporting.library.queries;
+
+public class BreastfeedingQueries {
+	
+	public static String getPatientsWhoGaveBirthTwoYearsAgo() {
+		return "select 	pg.patient_id" + " from patient p" + " inner join patient_program pg on p.patient_id=pg.patient_id"
+		        + " inner join patient_state ps on pg.patient_program_id=ps.patient_program_id"
+		        + " where pg.voided=0 and ps.voided=0 and p.voided=0 and"
+		        + " pg.program_id=8 and ps.state=27 and ps.end_date is null and"
+		        + " ps.start_date between date_add(:startDate, interval -2 year) and date_add(:startDate, interval -1 day) and location_id=:location";
+	}
+	
+	/**
+	 * GRAVIDAS INSCRITAS NO SERVIÇO TARV
+	 */
+	public static String getPregnantWhileOnArt(int concept1, int concept2, int concept3, int concept4, int enc1, int enc2,
```

Comment:
@ningosi Should this query maybe be in `PregnantQueries.java` file?

Also can we please use a more descriptive parameter name for concept1, concept2, etc... similar to how you did it in `getWeeksPregnantOnInitialOrFollowUpConsultations()` for example.

Reply:
I will move that to respective file and provide descriptive name parameters